pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java

```diff
@@ -1076,5 +1077,31 @@ public void setSchemaValidtionEnforced(@PathParam("tenant") String tenant,
         internalSetSchemaValidationEnforced(schemaValidationEnforced);
     }
 
+    @POST
+    @Path("/{tenant}/{namespace}/offloadPolicies")
+    @ApiOperation(value = " Set offload configuration on a namespace.")
+    @ApiResponses(value = {
+            @ApiResponse(code = 403, message = "Don't have admin permission"),
+            @ApiResponse(code = 404, message = "Namespace does not exist"),
+            @ApiResponse(code = 409, message = "Concurrent modification"),
+            @ApiResponse(code = 412, message = "Bucket must be specified") })
```

Comment:
412 is thrown for the following conditions:

- driver is not supported.
- bucket is not valid.
- offload policies is empty.

Reply:
I was so careless